content/docs/platform/workflow/digest.mdx

```diff
@@ -29,6 +29,139 @@ The time interval determines how long the digest engine will wait before sending
 
 Here, in the image below, `5` is the interval amount, and `mins` is the interval unit. Interval units can be `sec(s)`, `min(s)`, `hour(s)`, or `day(s)`.
 
+## Digest event count variables
+
+Events count variables provide a structured way to access, format, and display digest event data across different messaging channels in your workflow using built-in logic and formatting powered by LiquidJS filters. 
+
+Instead of manually writing custom logic to count events or format summary content, you can reference totals and relevant details directly within your notification templates to render digest summaries, such as daily digests or batch emails.
+
+<Callout type="info" title="Note">These variables are automatically exposed in all variable-supporting inputs following a digest step. They are not editable, as their values are determined by the digest payload. </Callout>
+
+### Digest variables
+
+Digest variables are exposed after a Digest Step has been executed and are available in all subsequent steps. These variables let you work directly with the collected events, including iterating through them or displaying a summary count.
+
+#### steps.digest-step.events
```

Comment:
The `digest-step` part is a dynamic part that is the digest step identifier. Usually, it's constant when you add the step, but can change if you add other digest steps and remove the previous. Maybe it could be worth mentioning that its dynamic part.